### PR TITLE
Ongoing: Naxx / OS / EoE / Various bug fixes / Various improvements

### PR DESCRIPTION
**Pull request by andrew6180**
Since this is a ton of changes made, I'll make a changelog here.

* **added Details_TinyThreat backport. Tested in dungeons & raids. references: https://github.com/Kowson/Details-Damage-Meter-for-3.3.5a/issues/2** 
* Added Naxxramas
* Added Eye of Eternity
* Added Obsidian Sanctum
* **Changed all encounter IDs to match the dungeon encounter ids found in the game files (https://wow.tools/dbc/?dbc=dungeonencounter&build=3.3.5.12340)**
* Tidy up VoA 
* Removed name limitations on NickTag (a common edit people did in legion)
* Added Naxx bosses as avatars for NickTag 
* **Reworked absorb healing tracking to mimic modern details a bit closer. Tested in raids on disc priest**
* Added a work around for https://github.com/Kowson/Details-Damage-Meter-for-3.3.5a/issues/3 by forcing capture to true on start combat @ https://github.com/ajseward/Details-Damage-Meter-for-3.3.5a/blob/678d4ad1189345e1bd91d2f50a02130e0b33733a/Details/core/control.lua#L319-L323 
* **Changed hardcoded `zoneMapID` finder to use `GetCurrentMapAreaID()` which in my limited testing, returned the correct ID.**
* Fixed up boss_faces / icon256x128 for toc / voa / ulduar 

*boss_heads folder can be dropped later, it's useless to the addon but useful for creating portraits*

### CODE REVIEW

File: Details/core/control.lua [hunk 1]
```diff
@@ -314,6 +316,16 @@
 			_table_wipe(_details.cache_healing_group)
 			_details:UpdateParserGears()
 			
+			if (not _details:CaptureGet("damaage")) then _details:CaptureSet(true, "damage", true) end
```
Comment (Kowson) [inline Details/core/control.lua]:
```suggestion
			if (not _details:CaptureGet("damage")) then _details:CaptureSet(true, "damage", true) end
```

Comment (andrew6180) [reply Details/core/control.lua]: fixed in https://github.com/ajseward/Details-Damage-Meter-for-3.3.5a/pull/1

**Comment by Zidras** [comment]
Hello! Sorry for just dropping this here as it is a bit unrelated but it may help. Since you are developing this backport even further, I have found another backport of a more recent version: https://gitlab.com/knights-of-sunwell/details-wotlk
(it's riddled with bugs and broken functions but since the original was a more recent version, it might have some optimization that could be helpful to you).

**Comment by Kowson** [comment]
Seems fine after a few days of testing.

**Pull request merged**